app/src/main/java/com/simply/birthdayapp/data/repositories/ResetPasswordEmailRepository.kt

```diff
@@ -0,0 +1,45 @@
+package com.simply.birthdayapp.data.repositories
```

Comment:
Please change file name from ResetPasswordEmailRepository to ResetPasswordRepository